infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/loader/dialect/datatypeloader/MySQLDataTypeLoader.java

```diff
@@ -46,6 +46,7 @@ public Map<String, Integer> load(final DatabaseMetaData database) throws SQLExce
         result.putIfAbsent("POLYGON", Types.BINARY);
         result.putIfAbsent("MULTIPOLYGON", Types.BINARY);
         result.putIfAbsent("MULTILINESTRING", Types.BINARY);
+        result.putIfAbsent("LINESTRING", Types.BINARY);
```

Comment:
1, It's better to put `LINESTRING` before `MULTILINESTRING`;

2, From the doc, looks there's `GEOMETRYCOLLECTION`, could it add here?

Reply:
OK.